macros/src/types/enum.rs

```diff
@@ -130,7 +134,7 @@ fn format_variant(
                     "{{ \"{}\": \"{}\", {} }}",
                     #tag,
                     #name,
-                    #inline_flattened
+                    #inline_flattened.trim_matches(&['{', '}', ' '])
```

Comment:
I think here, a comment would be usefull - How does `inline_flattened` look like, and why do we need to trim the braces here?

Reply:
Done!

Reply:
Basically, using [serde's example](https://serde.rs/enum-representations.html#internally-tagged), a type that should turn into this:
```ts
{"type": "Request", id: string, method: string, params: { /*...*/ } }
```

would turn into this
```ts
{"type": "Request", { id: string, method: string, params: { /*...*/ } } }
```